### ISSUE TITLE
Wrong value in types section of package.json for browser package

### ISSUE THREAD

**Issue by KleinMaximus** [pshrmn/hickory#60]
```
// /packages/browser/package.json
{
  "types": "types/index.d.ts"
}

```
But this file does not exist in the types folder

**Comment by pshrmn** [comment]
Ah, thanks! I'll get a patch published shortly.

**Comment by pshrmn** [comment]
I just published 2.0.1. Please let me know if there are any other issues.

**Comment by KleinMaximus** [comment]
Thanks for your job!
I posted a couple issues in `curi/svelte`
https://github.com/pshrmn/curi/issues
